proto/rill/ui/v1/dashboard.proto

```diff
@@ -7,11 +7,48 @@ import "rill/runtime/v1/queries.proto";
 
 // DashboardState represents the dashboard as seen by the user
 message DashboardState {
-  enum DashboardLeaderboardContextColumn {
-    DASHBOARD_LEADERBOARD_CONTEXT_COLUMN_UNSPECIFIED=0;
-    DASHBOARD_LEADERBOARD_CONTEXT_COLUMN_PERCENT = 1;
-    DASHBOARD_LEADERBOARD_CONTEXT_COLUMN_DELTA_CHANGE = 2;
-    DASHBOARD_LEADERBOARD_CONTEXT_COLUMN_HIDDEN = 3;
+  enum LeaderboardContextColumn {
+    LEADERBOARD_CONTEXT_COLUMN_UNSPECIFIED=0;
```

Comment:
@AdityaHegde -- can you figure out a way to remove the `_UNSPECIFIED` option on this and the other enums? This new options change the semantics of these enums -- in each case, the enums had previously enumerated all the valid states of the dashboard, but now an invalid state is included.

Reply:
Either that, or we might need separate enums to track the valid states in the URL data and in the actual dashboard, and/or a more sophisticated translation layer. I just don't want the `*_UNSPECIFIED` semantics to have any possibility of leaking into the actual product state